### PR TITLE
Resources setup

### PR DESCRIPTION
**Pull request by KovylinDenys**
Notes:
- To be reviewed after https://github.com/temyco/coffee-shop-kotlin/pull/1 merged 
- activity_main.xml changes would be reverted before merging since it only contains elements for the test/review purposes.

This PR adds:
- Colors
- Fonts
- Theme
- Icons (launcher & all product)
- Dimensions 
- Styles
- - Buttons
- - Radio Buttons
- - Checkbox

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                tools:text="RadioButton"/>
+
+            <androidx.appcompat.widget.AppCompatRadioButton
+                style="@style/CoffeeShop.RadioButton"
+                tools:text="RadioButton"/>
+        </RadioGroup>
+    </LinearLayout>
+
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (KovylinDenys) [inline app/src/main/res/layout/activity_main.xml]: Will be deleted before merge. Left for test/review purposes.

File: app/build.gradle [hunk 2]
```diff
@@ -9,7 +9,7 @@ android {
 
     defaultConfig {
         applicationId "com.temy.coffeeshop"
-        minSdkVersion 16
+        minSdkVersion 26
```
Comment (ababicheva) [inline app/build.gradle]: 23

File: app/src/main/res/drawable/button_highlighted_background.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ababicheva) [inline app/src/main/res/drawable/button_highlighted_background.xml]: sl_ 
use it for selectors

File: app/src/main/res/drawable/button_secondary_background.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@drawable/button_secondary_background_passive"
+        android:state_focused="false" />
+    <item
+        android:drawable="@drawable/button_secondary_background_tap"
+        android:state_focused="true" />
```
Comment (ababicheva) [inline app/src/main/res/drawable/button_secondary_background.xml]: state_focused -> state_pressed

File: app/src/main/res/drawable/ic_check.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="20dp"
+    android:height="20dp"
+    android:viewportWidth="20"
+    android:viewportHeight="20">
+  <path
+      android:pathData="M13.755,8.243c0.327,0.323 0.327,0.848 0,1.172l-3.254,3.22c-0.491,0.486 -1.286,0.486 -1.778,0l-1.478,-1.463c-0.327,-0.323 -0.327,-0.848 0,-1.172 0.327,-0.323 0.857,-0.323 1.183,0l0.59,0.585c0.328,0.324 0.86,0.324 1.188,0l2.365,-2.342c0.327,-0.324 0.857,-0.324 1.184,0z"
+      android:fillColor="#000"
+      android:fillType="evenOdd"/>
```
Comment (ababicheva) [inline app/src/main/res/drawable/ic_check.xml:9]: check that every vector drawable has fillType. it's a workaround for some Samsung devices.

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -1,18 +1,91 @@
...
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <LinearLayout
```
Comment (ababicheva) [inline app/src/main/res/layout/activity_main.xml]: REMOVE!!!

File: app/src/main/res/values/themes.xml [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
+        <item name="colorPrimary">@color/dark</item>
+        <item name="colorPrimaryVariant">@color/charcoal_grey</item>
         <item name="colorOnPrimary">@color/white</item>
         <!-- Secondary brand color. -->
-        <item name="colorSecondary">@color/teal_200</item>
-        <item name="colorSecondaryVariant">@color/teal_700</item>
-        <item name="colorOnSecondary">@color/black</item>
+        <item name="colorSecondary">@color/pale_grey</item>
+        <item name="colorSecondaryVariant">@color/white</item>
+        <item name="colorOnSecondary">@color/dark</item>
```
Comment (ababicheva) [inline app/src/main/res/values/themes.xml:11]: please add color accent, status bar -> white color

File: app/src/main/res/drawable/sl_button_outlined_big_background.xml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@drawable/button_outlined_big_background"
+        android:state_pressed="false" />
```
Comment (ababicheva) [inline app/src/main/res/drawable/sl_button_outlined_big_background.xml]: please remove state_pressed="false".
please check if items order is important is selectors.

File: app/src/main/res/values/styles.xml [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+    <style name="App.Button.Primary.Big" parent="@style/App.Button.Primary">
+        <item name="android:minHeight">@dimen/button_primary_big_height</item>
+    </style>
+
+    <style name="App.Button.Primary.Medium" parent="@style/App.Button.Primary">
+        <item name="android:minHeight">@dimen/button_primary_medium_height</item>
+    </style>
+
+    <style name="App.Button.Primary.Small" parent="@style/App.Button.Primary">
+        <item name="android:layout_width">wrap_content</item>
```
Comment (ababicheva) [inline app/src/main/res/values/styles.xml]: please remove width - we'll specify it in layout

File: app/src/main/res/values/styles.xml [hunk 10]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Buttons -->
+
+        <!-- Primary -->
+
+    <style name="App.Button.Primary" parent="@style/Widget.AppCompat.Button.Colored">
+        <item name="android:layout_width">match_parent</item>
```
Comment (ababicheva) [inline app/src/main/res/values/styles.xml]: remove layout_witdh and layout_height from all styles where it hasn't be specified by design

File: app/src/main/res/values/styles.xml [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+    <style name="App.Button.Outlined.Tiny" parent="@style/App.Button.Outlined">
+        <item name="android:layout_height">@dimen/button_outlined_tiny_height</item>
+        <item name="android:background">@drawable/sl_button_outlined_tiny_background</item>
+        <item name="android:textAppearance">@style/Button500MColorOrangered</item>
+        <item name="android:textAllCaps">true</item>
+    </style>
+
+        <!-- Check box -->
+
+    <style name="CoffeeShop.CheckBox" parent="@style/Widget.AppCompat.CompoundButton.CheckBox">
```
Comment (ababicheva) [inline app/src/main/res/values/styles.xml]: App.Checkbox

**Review by ababicheva — APPROVED** [review]

**Pull request merged**